app/models/minion.rb

```diff
@@ -25,7 +25,8 @@ def self.update_grains
 
         tx_update_reboot_needed = minion_grains["tx_update_reboot_needed"] || false
         tx_update_failed = minion_grains["tx_update_failed"] || false
-        minion.update_columns online:                  online,
+        minion.update_columns fqdn:                    minion_grains["fqdn"],
+                              online:                  online,
```

Comment:
Is the reconciliation loop going to keep this grain in sync with the database?

Reply:
@flavio yes it does. it [calls exactly this method here](https://github.com/kubic-project/velum/blob/master/lib/tasks/salt.rake#L18)